x/wasm/internal/keeper/staking_test.go

```diff
@@ -380,8 +373,8 @@ func TestReinvest(t *testing.T) {
 
 	// check some account values - the money is on neither account (cuz it is bonded)
 	// Note: why is this immediate? just test setup?
-	checkAccount(t, ctx, accKeeper, contractAddr, sdk.Coins{})
-	checkAccount(t, ctx, accKeeper, bob, funds)
+	checkAccount(t, ctx, accKeeper, bankKeeper, contractAddr, sdk.Coins{})
```

Comment:
Nice that so much of this code "just works" without changes. And the contracts as well.